### DISCUSSION TITLE
Add example to DuckDB shell .help

### DISCUSSION THREAD

**Discussion by Alex-Monahan** [duckdb/duckdb-wasm#326] [Ideas]
It's a tiny thing, but maybe a Select query from the blog post that points at one of the files that are hosted on github could be added to the .help output in the shell? It wasn't easy for me to understand how to get started.

select * from 'https://shell.duckdb.org/data/tpch/0_01/parquet/orders.parquet' limit 10; 

Thanks!

**Comment by ankoh** [comment]
Yes, I agree. We should add a dedicated `.examples` command. https://github.com/duckdb/duckdb-wasm/issues/256

**Reply by luceydav** [reply]
I have spent bit of time trying to install and load the httfs extension in DuckDB-WASM, because it seems to install, but fails silently. Also, this .examples does load a file from https, further confusing me. I eventually found the DuckDB-WASM HTTPFS extension is not yet available in the sub-section for extensions in the DuckDB-WASM. I suggest an error message after install httpfs pointing this out. I apologize if I am not understanding something here.

**Reply by carlopi** [reply]
Thanks for the feedback!
Plan is support httpfs (of some part of it) explicitly as part of duckdb-wasm, I will have to think a bit what's the right way but more feedback to users is always good.
